.github/workflows/testing.yml

```
@@ -23,7 +23,30 @@ jobs:
   test-action: # make sure the action works on a clean machine without building
       runs-on: ubuntu-latest
       steps:
-        - uses: actions/checkout@v2
-        - uses: ./
-        - name: Check that result file is generated
-          run: ls -l out/analyzer-result.yml
+        - name: Clone this repository
+          uses: actions/checkout@v2
+
+
+        - name: test without any input arguments
+          uses: ./
+        - name: Check the resulting directory tree
+          run: tree -a -L 5 -I '.git' .
+
+        - name: thename
```

Can we make 2 separate jobs so tests dont interfere with each other.
```suggestion
  test-action-multiple: 
    name: Test action with multiple repositories
    runs-on: ubuntu-latest
    steps:
        - name: thename
```

---

yep still working on it